app/main.py

```diff
@@ -1 +1,64 @@
-# write your code here
+import os
+import shutil
+
+
+class Errors(BaseException):
+    pass
+
+
+class WrongCommandEr(Errors):
+    pass
+
+
+class WrongTypeEr(Errors):
+    pass
+
+
+class WrongCommandStructureEr(Errors):
+    pass
+
+
+class IsNotFileEr(Errors):
+    pass
+
+
+class NoSuchFile(Errors):
+    pass
+
+
+def is_smth_wrong_command_right(command: str) -> None:
+    if not isinstance(command, str):
```

Comment:
This is too much spaghetti. Your input validation should be like:
1. Check if you received exactly 3 arguments.
2. Check if comand is 'mv'.
3. After this you can safely split your command into 3 pieces.
4. Then you use `os.path.join/split` methods to get destination path.
5. Use `os.makedirs` to create dirs and `shutil.move` for main logic - wrap them in exceptions to handle wrong dir/file names and handle non-exsiting files.
6. This whole thing shouldn't take you more then 15 lines of code to write